examples/sonoff_basic/main.c

```diff
@@ -69,7 +71,14 @@ void switch_on_callback(homekit_characteristic_t *_ch, homekit_value_t on, void
 }
 
 void button_intr_callback(uint8_t gpio) {
-    if (gpio_read(button_gpio) == 1) {
+    uint32_t now = xTaskGetTickCountFromISR();
+    uint16_t debounce_time = 100;
```

Comment:
You can divide debounce_time by portTICK_PERIOD_MS (call it `const` or `static`) and save on multiplication.